src/libraries/System.Security.Cryptography/src/System/Security/Cryptography/X509Certificates/OpenSslCertificateAssetDownloader.cs

```diff
@@ -31,6 +31,7 @@ internal static class OpenSslCertificateAssetDownloader
             }
             catch (CryptographicException)
             {
+                OpenSslX509ChainEventSource.Log.InvalidDownloadedCertificate();
```

Comment:
Would we want to do the same for the early out where `Length = 0`?

EDIT: Ah, well, further down in `DownloadAsset` I see the `Length` is being logged... and `Length = 0` is "interesting", so at least that is captured.